docs/share.md

```diff
@@ -0,0 +1,82 @@
+# Share (Manila)
```

Comment:
I believe the generic component name is "Shared Filed System" not just "Share". 
Please change the name accordingly everywhere it needs to in order to reflect the full name, including in the file: `shared_file_system.md`. Thanks